packages/components/src/Combobox/components/ComboboxContent/ComboboxContentSearch/ComboboxContentSearch.css

```diff
@@ -12,7 +12,19 @@
   font-size: var(--typography--fontSize-base);
 }
 
+.searchInput::-webkit-search-decoration,
+.searchInput::-webkit-search-cancel-button,
+.searchInput::-webkit-search-results-button,
+.searchInput::-webkit-search-results-decoration {
+  -webkit-appearance: none;
```

Comment:
It looks like Storybook just uses `appearance: none` but this [solution I found](https://stackoverflow.com/questions/9421551/how-do-i-remove-all-default-webkit-search-field-styling) uses the webkit approach - should I use both? Will our post-processors add any webkit prefixes that are needed?

Reply:
Further down the page on your link they also suggest an `appearance` approach in 2023, but I did want to try your solution in all three browsers and it looks like it's working just fine as-is.

I think what might be making it 'work' better though is your `outline:none` below which in other circumstances would _technically_ interfere with accessibility, but we do our own `focus` state so we can safely disable the built-in browser solution without affecting anything negatively. 

Even React themselves supports that decision:
![image](https://github.com/GetJobber/atlantis/assets/145565743/55016185-28b6-4448-830b-c51d2ca4b0d3)